docs/modules/ROOT/pages/documentation/payara-server/upgrade-server.adoc

```diff
@@ -0,0 +1,59 @@
+[[Upgrade Server]]
+= Upgrade Payara Server
+
+Payara Server can be upgraded to new versions from version *5.24.0*
```

Comment:
This tool can be used to upgrade a Payara Server installation to a newer version. The Upgrade Tool is bundled in Payara Server versions from version *5.24.0*. It can also be obtained from Payara Nexus at https://nexus.payara.fish/#browse/browse:payara-enterprise:fish%2Fpayara%2Fextras%2Fpayara-upgrade-tool (
  groupId: `fish.payara.extras`, artifactId: `payara-upgrade-tool`). If obtained from the Payara Nexus, drop the JAR artifact into your Payara Server installation under `payara5/glassfish/lib/asadmin`. The version 1.0 of the standalone upgrade tool can be installed into Payara Server Enterprise 5.20.0 or newer and supports upgrading to any newer Payara Server Enterprise version.